<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>272</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #32323b;
border-radius:100px;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">border-radius:10px;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="minimizeBtn">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton::hover{
	background-color: rgb(72, 72, 87);
color:white;
border:none;
border-radius:0px;
}QPushButton{
	background-color: transparent;
color:white;
border:none;
}
</string>
        </property>
        <property name="text">
         <string></string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeBtn">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover {
	background-color: rgb(165, 0, 0);
	border:0px transparent;
border-radius:0px;
border-top-right-radius:10px!important;

}
QPushButton {
	background-color: transparent;
color:white;
}</string>
        </property>
        <property name="text">
         <string></string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <item row="2" column="0">
       <widget class="QPushButton" name="Bard">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	color: #E5E5EA;
	border: 1px solid rgba(255, 255, 255, 0.096);
    border-radius: 10px;
	padding-right: 2px;
	background-color:#32323b;

}

QPushButton:hover {
	background-color: rgba(255,255,255,5);
}

QPushButton:pressed {
	background-color: rgba(255,255,255,2);
}</string>
        </property>
        <property name="text">
         <string>    Bard</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../imgs/Bard.png</normaloff>../imgs/Bard.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="ShadowHunter">
        <property name="minimumSize">
         <size>
          <width>163</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	color: #E5E5EA;
	border: 1px solid rgba(255, 255, 255, 0.096);
    border-radius: 10px;
	padding-right: 2px;
	background-color:#32323b;

}

QPushButton:hover {
	background-color: rgba(255,255,255,5);
}

QPushButton:pressed {
	background-color: rgba(255,255,255,2);
}</string>
        </property>
        <property name="text">
         <string>    Shadow Hunter</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../imgs/ShadowHunter.png</normaloff>../imgs/ShadowHunter.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="Dracolyte">
        <property name="minimumSize">
         <size>
          <width>163</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	color: #E5E5EA;
	border: 1px solid rgba(255, 255, 255, 0.096);
    border-radius: 10px;
	padding-right: 2px;
	background-color:#32323b;

}

QPushButton:hover {
	background-color: rgba(255,255,255,5);
}

QPushButton:pressed {
	background-color: rgba(255,255,255,2);
}</string>
        </property>
        <property name="text">
         <string>   Dracolyte</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../imgs/Dracolyte.png</normaloff>../imgs/Dracolyte.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="IceSage">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	color: #E5E5EA;
	border: 1px solid rgba(255, 255, 255, 0.096);
    border-radius: 10px;
	padding-right: 2px;
	background-color:#32323b;

}

QPushButton:hover {
	background-color: rgba(255,255,255,5);
}

QPushButton:pressed {
	background-color: rgba(255,255,255,2);
}</string>
        </property>
        <property name="text">
         <string>    Ice Sage</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../imgs/IceSage.png</normaloff>../imgs/IceSage.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="Boomeranger">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	color: #E5E5EA;
	border: 1px solid rgba(255, 255, 255, 0.096);
    border-radius: 10px;
	padding-right: 2px;
	background-color:#32323b;

}

QPushButton:hover {
	background-color: rgba(255,255,255,5);
}

QPushButton:pressed {
	background-color: rgba(255,255,255,2);
}</string>
        </property>
        <property name="text">
         <string>    Boomeranger</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../imgs/Boomeranger.png</normaloff>../imgs/Boomeranger.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="Solarion">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	color: #E5E5EA;
	border: 1px solid rgba(255, 255, 255, 0.096);
    border-radius: 10px;
	padding-right: 2px;
	background-color:#32323b;
}

QPushButton:hover {
	background-color: rgba(255,255,255,5);
}

QPushButton:pressed {
	background-color: rgba(255,255,255,2);
}</string>
        </property>
        <property name="text">
         <string>  Solarion</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../imgs/Solarion.png</normaloff>../imgs/Solarion.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="Savebtn">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>25</height>
         </size>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	color: #E5E5EA;
	border: 1px solid rgba(255, 255, 255, 0.096);
    border-radius: 5px;
	padding: 4px;
}

QPushButton:hover {
	background-color: rgba(255,255,255,5);
}

QPushButton:pressed {
	background-color: rgba(255,255,255,2);
}
</string>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
